src/Modals/ModalNewForm/ModalNewForm.css

```diff
@@ -71,3 +66,29 @@ input{
     font-weight: bolder;
 }
 
+.button_submit, .button_cancel{
```

Comment:
sometimes it's better to just have multiple classes applied to divs. For example, in this instance, it seems you want a generic `.button` class, and then `.button_cancel` and `.button_submit` can be more specific, and the div for button_cancel would have both `.button .button_cancel` classes